site/pages/about-us/principles-slice/style.css

```diff
@@ -31,9 +31,14 @@
   padding-bottom: 50px;
 }
 
-.buttons a button{
-  margin-bottom: 20px;
+.link{
```

Comment:
@dstodolny Thanks for the review, but can't think of a good name for them. There are other styles in the component that are just as generic - like the buttons container that they're in